src/index.js

```diff
@@ -14,4 +13,3 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
```

Comment:
You also need to remove this. you know why? If you don't I can explain.

Reply:
I don't know.please explain to me.